extensions/amp-story/1.0/amp-story.css

```diff
@@ -25,12 +25,18 @@
 @import './pagination-buttons.css';
 
 /** Common */
-amp-story, amp-story-page, amp-story-grid-layer, amp-story-cta-layer {
+amp-story, amp-story-page, amp-story-cta-layer {
   contain: strict !important;
   overflow: hidden !important;
   -webkit-touch-callout: none !important;
 }
 
+amp-story-grid-layer {
```

Comment:
Is there still some `contain` value we could/should use here?

Reply:
In theory `contain: layout size` can still be used. I added it.